code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -0,0 +1,96 @@
+/obj/item/living_heart
+	name = "Living Heart"
+	desc = "Link to the worlds beyond."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "living_heart"
+	var/datum/mind/target
+
+/obj/item/living_heart/attack_self(mob/user)
+	. = ..()
+	if(!isecultist(user))
+		return
+	if(!target)
+		return
+	var/dist = get_dist(user,target)
+
+	switch(dist)
+		if(0 to 5)
+			to_chat(user,"<span class='warning'>[target.current.real_name] is near you</span>")
+		if(6 to 15)
+			to_chat(user,"<span class='warning'>[target.current.real_name] is somewhere in your vicinty</span>")
+		if(16 to 64)
+			to_chat(user,"<span class='warning'>[target.current.real_name] is far away from you</span>")
+		else
+			to_chat(user,"<span class='warning'>[target.current.real_name] is beyond our reach</span>")
+
+	if(target.current.stat == DEAD)
+		to_chat(user,"<span class='warning'>[target.current.real_name] is dead. Bring him onto the transmutation rune!</span>")
+
+/obj/item/melee/sickly_blade
+	name = "Sickly blade"
+	desc = "Twisted sickle with an ornamental eye. The eyes looks at you."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "eldritch_blade"
+	item_state = "eldritch_blade"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	flags_1 = CONDUCT_1
+	sharpness = IS_SHARP
+	w_class = WEIGHT_CLASS_BULKY
+	force = 15
+	throwforce = 10
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "rended")
+
+/obj/item/melee/sickly_blade/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	var/datum/antagonist/ecult/cultie = user.mind.has_antag_datum(/datum/antagonist/ecult)
+	if(!cultie)
+		return
+	for(var/X in cultie.get_all_knowledge())
+		var/datum/eldritch_knowledge/EK = X
+		EK.eldritch_blade_act(target,user,proximity_flag,click_parameters)
+
+/obj/item/melee/sickly_blade/rust
+	name = "Rusted Blade"
+
+/obj/item/melee/sickly_blade/ash
+	name = "Ashen Blade"
```

Comment:
proposal: make the ashen version (and only the ashen version, since the ashies are supposed to be the stealthy/sneaky cultists) of the sickly blade smaller, so that you can carry it around without wearing a big suit of armor that screams "I'M A CULTIST"

or don't do that if it'd break the balance of eldritch cultists

Reply:
that would most likely break the balance, since it is a path of exile style skill tree.